packages/components/nodes/memory/BufferWindowMemory/BufferWindowMemory.ts

```diff
@@ -0,0 +1,61 @@
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
+import { getBaseClasses } from '../../../src/utils'
+import { BufferWindowMemory, BufferWindowMemoryInput } from 'langchain/memory'
+
+class BufferWindowMemory_Memory implements INode {
+    label: string
+    name: string
+    description: string
+    type: string
+    icon: string
+    category: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+
+    constructor() {
+        this.label = 'Buffer Window Memory'
+        this.name = 'bufferWindowMemory'
+        this.type = 'BufferWindowMemory'
+        this.icon = 'memory.svg'
+        this.category = 'Memory'
+        this.description = 'Uses a window of size k to surface the last k back-and-forths to use as memory'
+        this.baseClasses = [this.type, ...getBaseClasses(BufferWindowMemory)]
+        this.inputs = [
+            {
+                label: 'Memory Key',
+                name: 'memoryKey',
+                type: 'string',
+                default: 'chat_history'
+            },
+            {
+                label: 'Input Key',
+                name: 'inputKey',
+                type: 'string',
+                default: 'input'
+            },
+            {
+                label: 'Size',
+                name: 'k',
+                type: 'number',
+                default: '4'
```

Comment:
can add a `description: 'window of size k to surface the last k back-and-forths to use as memory.'` so people can understand whats this k for